tests/NRedisStack.Tests/Graph/GraphTests.cs

```diff
@@ -956,6 +956,49 @@ public void TestModulePrefixs()
         Assert.NotEqual(graph1.GetHashCode(), graph2.GetHashCode());
     }
 
+    [Fact]
+    public void TestCallProcedureDbLabels()
+    {
+        var db = redisFixture.Redis.GetDatabase();
+        db.Execute("FLUSHALL");
+
+        const string graphName = "social";
+
+        var graph = db.GRAPH();
+        // Create empty graph, otherwise call procedure will throw exception
+        graph.Query(graphName, "RETURN 1"); 
```

Comment:
trailing space, It's worth checking the rest of the code as well and making sure there are no more trailing spaces.

Reply:
I can find trailing whitespace in `src/NRedisStack/ResponseParser.cs`,  `tests/Doc/SearchQuickStartExample.cs` and `tests/Doc/SetGetExample.cs`.
Do you want me to trim those as well or create a separate PR that is more inline with #116?

Reply:
I think it's better separately to keep the PR clean.
But in the code of this PR there should be no trailing spaces.